src/components/Nav/NavList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import NavItem from './NavItem';
+
+const NavList = props => {
+  const { category, list } = props;
```

Comment:
- 함수의 매개변수로 들어오는 값이 객체 또는 배열일 경우 매개변수자리에서 바로 구조분해할당이 가능합니다.
- 아래코드를 참고해보세요
```suggestion
const NavList = ({ category, list }) => {
```